scripts/update-flake.sh

```diff
@@ -13,4 +27,14 @@ echo "Calculating SRI hash..."
 HASH=$(go run tailscale.com/cmd/nardump --sri "$OUT/pkg/mod/cache/download")
 sudo rm -rf "$OUT"
 
+echo "Updating go.mod vendorHash"
 sed -i "s#\(vendorHash = \"\)[^\"]*#\1${HASH}#" ./flake.nix
+
+# Update protoc-gen-go sha256
+echo "Updating protoc-gen-go sha256..."
+PROTOC_GEN_GO_REV=$(nix eval --extra-experimental-features nix-command --extra-experimental-features flakes --raw .#proto_gen_go.rev)
+echo "protoc-gen-go version: $PROTOC_GEN_GO_REV"
+PROTOC_GEN_GO_SHA256=$(nix-prefetch-git https://github.com/protocolbuffers/protobuf-go --rev "$PROTOC_GEN_GO_REV" | jq -r .hash)
```

Comment:
@joobisb can we not use the `nix-command flakes` approach for this? My previous comment will apply to this, too (about compatibility).

Reply:
Actually this is fine, I think.